types/karma/index.d.ts

```diff
@@ -217,7 +293,7 @@ export interface ConfigOptions {
      * You can capture any browser manually by opening the browser and visiting the URL where
      * the Karma web server is listening (by default it is <code>http://localhost:9876/</code>).
      */
-    browsers?: string[];
+    browsers?: Array<AutomatedBrowsers | string>;
```

Comment:
TypeScript will normalise this to `string[]`, so this is only useful for documentation. See https://github.com/microsoft/TypeScript/issues/12687.

Reply:
I know, the intent here is to list those supported types in the source code so we could poke around without reading comments and also export the AutomatedBrowser type - because that's valid and it supports the compiler/intellisense. Aligned with that:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42208#discussion_r377217978
I agree with your comment, but saying above, let me know if I should amend this, thx!

Reply:
Not really, was just FYI. 
I agree that having it for documentation purposes is useful on its own :+1: